### PR TITLE
feat: twitter blacklist commands

### PR DESCRIPTION
**Pull request by anhnh12**
**What does this PR do?**
New twitter blacklist commands: `$poe twitter block <user>` or `$poe twitter block <action>`
-   [x] Block a user: `$poe twitter block <@username or ID>`
-   [x] Unblock a user: `$poe twitter block remove <@username or ID>`
-   [x] Show blacklist: `$poe twitter block list`

**Media**
image
image
image

### CODE REVIEW

File: src/commands/config/poe/twitter/block/index.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      return {
+        messageOptions: {
+          embeds: [
+            getErrorEmbed({
+              msg,
+              description: "This command must be run in a Guild",
+            }),
+          ],
+        },
+      }
```
Comment (tuanddd) [inline src/commands/config/poe/twitter/block/index.ts:42]: use `GuildIdNotFoundError`

File: src/commands/config/poe/twitter/block/list.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      return {
+        messageOptions: {
+          embeds: [
+            getErrorEmbed({
+              msg,
+              description: "This command must be run in a Guild",
+            }),
+          ],
+        },
+      }
```
Comment (tuanddd) [inline src/commands/config/poe/twitter/block/list.ts:25]: same

File: src/commands/config/poe/twitter/block/list.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      description,
+    })
+
+    return { messageOptions: { embeds: [embed] } }
+  },
+  getHelpMessage: async (msg) => ({
+    embeds: [
+      composeEmbedMessage(msg, {
+        usage: `${PREFIX}poe twitter block <twitter ID or username>\n${PREFIX}poe twitter block <action>`,
+        examples: `${PREFIX}poe twitter block @randomeuser\n${PREFIX}poe twitter block 123123123123`,
```
Comment (tuanddd) [inline src/commands/config/poe/twitter/block/list.ts:50]: wrong content

File: src/commands/config/poe/twitter/block/remove.ts [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          }),
+        ],
+      },
+    }
+  },
+  getHelpMessage: async (msg) => ({
+    embeds: [
+      composeEmbedMessage(msg, {
+        usage: `${PREFIX}poe twitter block remove <twitter ID or username>`,
+        examples: `${PREFIX}poe twitter block remove @randomeuser\n${PREFIX}poe twitter block remove 123123123123`,
```
Comment (tuanddd) [inline src/commands/config/poe/twitter/block/remove.ts:71]: wrong content

File: src/commands/config/poe/twitter/block/remove.ts [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+      return {
+        messageOptions: {
+          embeds: [
+            getErrorEmbed({
+              msg,
+              description: "This command must be run in a Guild",
+            }),
+          ],
+        },
+      }
```
Comment (tuanddd) [inline src/commands/config/poe/twitter/block/remove.ts:31]: same

**Review by tuanddd — CHANGES_REQUESTED** [review]

**Review by catngh — APPROVED** [review]

File: src/commands/config/poe/twitter/block/remove.ts (hunk 4 above)
Comment (anhnh12) [reply src/commands/config/poe/twitter/block/remove.ts:55]: why wrong sir?

**Review by tuanddd — APPROVED** [review]

**Pull request merged**